patches/content-public-browser-web_contents_receiver_set.h.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/content/public/browser/web_contents_receiver_set.h b/content/public/browser/web_contents_receiver_set.h
+index c373078ea48ebf2886b3a42513eb795d24da159d..8cf8525f3a3b5722b39766d8400854ba78284858 100644
+--- a/content/public/browser/web_contents_receiver_set.h
++++ b/content/public/browser/web_contents_receiver_set.h
+@@ -59,6 +59,11 @@ namespace subresource_filter {
+ class ContentSubresourceFilterThrottleManager;
+ }
+ 
++class BraveDrmTabHelper;
```

Comment:
this is definitely not the right way to patch this, is this temporary or something?

Reply:
@bridiver You're right on this was meant to be a temporary thing and needs to be fixed. At the time I fixed the issue I read the threads [in the CL](review.googlesource.com/c/chromium/src/+/2956368) about how this was meant to be temporary and there were plans to fix them progressively, but there were no migrations done yet at the time (I wrote this on June 23rd, and the [first migration upstream is from July 14th](https://bugs.chromium.org/p/chromium/issues/detail?id=1213679#c3)).

Then I somehow forgot to check again once I was back from holidays and now it's not only that there are plenty of migrations done upstream, but also that even [WebContentsReceiverSet has been removed](https://bugs.chromium.org/p/chromium/issues/detail?id=1213679#c28)!

I'll work on a fix for this now, thanks for catching this!

Reply:
Fixed on https://github.com/brave/brave-core/commit/c698321ba83e0fdec507e3c3627c02f70a5afbda and https://github.com/brave/brave-core/commit/57354fecd87bcd8f1936b1a7dca5276009a53a0a